### PR TITLE
Making artisan fresh more secure

### PR DESCRIPTION
**Pull request by adamgoose**
I can imagine this getting ran accidentally after the project's been on for a good while...

### CODE REVIEW

**Comment by RomainLanz** [comment]
:+1:

File: src/Illuminate/Foundation/Console/FreshCommand.php [hunk 1]
```diff
@@ -28,6 +28,12 @@ public function fire()
 	{
 		$files = new Filesystem;
 
+		if($files->exists($this->laravel->storagePath().'/framework/fresh'))
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Console/FreshCommand.php]: missing space

Comment (adamgoose) [inline src/Illuminate/Foundation/Console/FreshCommand.php]: Addressed.

**Comment by taylorotwell** [comment]
I would like a better way to know it has been run really. Also, stuff should be in source control :)

**Comment by adamgoose** [comment]
Figured as much. This is all I could come up with in the 30 seconds I had to submit it.

Could we set a config option? Set a flag on the console Kernel?

**Pull request closed without merging**